### PR TITLE
Make INSTALL_DIR overridable

### PR DESCRIPTION
**Pull request by peterkrenn**
So I can run `INSTALL_DIR=~/.bin ./install.sh`

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -23,12 +28,14 @@ if [ -f "$LOCAL_JAR_FILE" ]; then
 	install "$LOCAL_JAR_FILE" "$INSTALL_DIR"
 else
 	echo "$LOCAL_JAR_FILE does not exist, pulling latest release from Github."
-	curl -L https://github.com/platogo/atoss-cli/releases/latest/download/atoss-cli-standalone.jar >atoss-cli-standalone.jar
-	install atoss-cli-standalone.jar $INSTALL_DIR
+	mkdir -p $LOCAL_JAR_PATH
+	curl -L https://github.com/platogo/atoss-cli/releases/latest/download/atoss-cli-standalone.jar > $LOCAL_JAR_FILE
```
Comment (DaniruKun) [inline install.sh:32]:
```suggestion
	curl -L https://github.com/platogo/atoss-cli/releases/latest/download/atoss-cli-standalone.jar > "$LOCAL_JAR_FILE"
```

File: install.sh [hunk 2]
```diff
@@ -23,12 +28,14 @@ if [ -f "$LOCAL_JAR_FILE" ]; then
 	install "$LOCAL_JAR_FILE" "$INSTALL_DIR"
 else
 	echo "$LOCAL_JAR_FILE does not exist, pulling latest release from Github."
-	curl -L https://github.com/platogo/atoss-cli/releases/latest/download/atoss-cli-standalone.jar >atoss-cli-standalone.jar
-	install atoss-cli-standalone.jar $INSTALL_DIR
+	mkdir -p $LOCAL_JAR_PATH
```
Comment (DaniruKun) [inline install.sh:31]:
```suggestion
	mkdir -p "$LOCAL_JAR_PATH"
```

File: install.sh [hunk 3]
```diff
@@ -23,12 +28,14 @@ if [ -f "$LOCAL_JAR_FILE" ]; then
...
+	curl -L https://github.com/platogo/atoss-cli/releases/latest/download/atoss-cli-standalone.jar > $LOCAL_JAR_FILE
+	install $LOCAL_JAR_FILE $INSTALL_DIR
```
Comment (DaniruKun) [inline install.sh:33]:
```suggestion
	install "$LOCAL_JAR_FILE" $INSTALL_DIR
```

**Review by DaniruKun — CHANGES_REQUESTED** [review]

**Comment by peterkrenn** [comment]
Sorry, I missed that. Otherwise I'd have applied the udpates.

**Pull request merged**